utils/build-ios-framework.sh

```diff
@@ -10,9 +10,10 @@ if [ ! -d destroot/Library/Frameworks/iphoneos/hermes.framework ]; then
     ios_deployment_target=$(get_ios_deployment_target)
 
     build_apple_framework "iphoneos" "armv7;armv7s;arm64" "$ios_deployment_target"
-    build_apple_framework "iphonesimulator" "x86_64;i386" "$ios_deployment_target"
+    build_apple_framework "iphonesimulator" "x86_64;arm64" "$ios_deployment_target"
```

Comment:
BTW, I do think we should brig `i386` back. We still support iOS 10.

Ref: https://github.com/facebook/hermes/pull/363#issuecomment-696626229